src/arrayMethodSort.js

```diff
@@ -3,10 +3,39 @@
 /**
  * Implement method Sort
  */
+const compareAsString = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA < stringB) {
+    return -1;
+  } else {
+    return 0;
+  }
```

Comment:
Last else is redundant
```suggestion
  }
  
  return 0;
```